pulp_rpm/tests/functional/api/test_sync.py

```diff
@@ -1122,10 +1123,13 @@ def optimize_report(self, progress_reports=[]):
                 return True
         return False
 
+    @unittest.skip(
+        "Needs a repo where an unacceptable checksum is used for packages, but not for metadata"
```

Comment:
I'm now checking the checksums of the metadata when downloading it, so we fail when downloading the metadata rather than when downloading packages.